{"name": "test_s1", "status": "broken", "statusDetails": {"message": "NameError", "trace": "open = None\n\n    def test_s1(open):\n        print(\"用例1：搜索python-1\")\n    \n        # 如果第一个用例异常了，不影响其他的用例执行\n>       raise NameError  # 模拟异常\nE       NameError\n\ntest_yield2.py:16: NameError"}, "start": 1676887788918, "stop": 1676887788918, "uuid": "0734af7e-563f-4339-9118-9d0562fa4c90", "historyId": "9ac642efa5c0e038221bfb1b5c15224c", "testCaseId": "2430cd747812565ef7f63f1b40ce3ec3", "fullName": "test_yield2#test_s1", "labels": [{"name": "suite", "value": "test_yield2"}, {"name": "host", "value": "DESKTOP-KG3DMFK"}, {"name": "thread", "value": "22308-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_yield2"}]}